packages/flutter/lib/src/material/about.dart

```diff
@@ -1065,7 +1064,7 @@ class _MasterDetailFlow extends StatefulWidget {
 
   /// Builder for the master view for lateral navigation.
   ///
-  /// If [masterPageBuilder] is not supplied the master page required for nested navigation, also
+  /// This builder builds the master page required for nested navigation, also
```

Comment:
It appears the other parameter was never used, and it was removed in https://github.com/flutter/flutter/pull/96553